---
about_this_resource_text: <p><strong>Topics covered:</strong> Metals in Biology</p><p><strong>Instructor:</strong>
  Prof. Christopher Cummins</p><p><em>Audio only is available for this session.</em></p>
course_id: 5-112-principles-of-chemical-science-fall-2005
embedded_media:
- id: Audio-iTunesU-MP3
  media_location: ' https://itunes.apple.com/us/itunes-u/id341596934'
  parent_uid: e23e92da4e7decf55b9a4e6c8cba8d01
  title: Audio-iTunes U-MP3
  type: Audio
  uid: c611e95473b74f75867d149c47f299ce
- id: Audio-InternetArchive-MP3
  media_location: http://www.archive.org/download/MIT5.112F05/ocw-5.112-12dec2005.mp3
  parent_uid: e23e92da4e7decf55b9a4e6c8cba8d01
  title: Audio-Internet Archive-MP3
  type: Audio
  uid: db0a18f0aafa9e9444156afcce569cc9
- id: 3Play-3PlayVideoid-SRT
  media_location: '574875'
  parent_uid: e23e92da4e7decf55b9a4e6c8cba8d01
  title: 3Play-3Play Video id
  type: 3Play
  uid: 1d595414ec2009ae066c64ab116ae9b5
- id: 574875.srt
  parent_uid: e23e92da4e7decf55b9a4e6c8cba8d01
  technical_location: https://ocw.mit.edu/courses/chemistry/5-112-principles-of-chemical-science-fall-2005/video-lectures/lecture-35-metals-in-biology/574875.srt
  title: 3play caption file
  type: null
  uid: f729aa26f5f600090ffdf3960044f2b4
- id: 574875.pdf
  parent_uid: e23e92da4e7decf55b9a4e6c8cba8d01
  technical_location: https://ocw.mit.edu/courses/chemistry/5-112-principles-of-chemical-science-fall-2005/video-lectures/lecture-35-metals-in-biology/574875.pdf
  title: 3play pdf file
  type: null
  uid: 56be94d7daca180659c5b38d3bc483f7
- id: Caption-3Play Video id-SRT
  parent_uid: e23e92da4e7decf55b9a4e6c8cba8d01
  title: Caption-3Play Video id-SRT-English - US
  type: Caption
  uid: 0d3e916efff840da21e4740a07208848
- id: Transcript-3Play Video id-PDF
  parent_uid: e23e92da4e7decf55b9a4e6c8cba8d01
  title: Transcript-3Play Video id-PDF-English - US
  type: Transcript
  uid: 656dbe97a6a1a50c2e9cfef9f744f8ad
inline_embed_id: 57762794lecture35:metalsinbiology77926444
layout: video
order_index: null
parent_uid: 0f6eafa33e9056ec6097f69475356be6
related_resources_text: <p><a href="./resolveuid/9d5e3c3a9f278ed1cffcea06a6cda240">Metals
  in Biology (PDF)</a></p>
short_url: lecture-35-metals-in-biology
technical_location: https://ocw.mit.edu/courses/chemistry/5-112-principles-of-chemical-science-fall-2005/video-lectures/lecture-35-metals-in-biology
template_type: Tabbed
title: 'Lecture 35: Metals in Biology'
transcript: '<p>The following content is provided by MIT OpenCourseWare under a Creative
  Commons license. Additional information about our license and MIT OpenCourseWare
  in general is available at ocw.mit.edu.</p><p>PROFESSOR: Today, we''re going to
  talk about a very exciting subject regarding molecules. And this subject has to
  do with metals in biology. Metals in biology, by the way, is actually the name of
  one of the Gordon research conferences. So if you get excited about this topic,
  you can go to the Gordon research conference web page and apply to attend the conference
  next summer and learn about the latest developments in metals in biology.</p><p>One
  of the things I want to impart to you today has to do with where we usually find
  metals inside of biomolecules. So that leads me to talk a little bit about nature''s
  ligands. In biological chemistry, we find that molecules sometimes are quite a bit
  larger than we''re used to looking at in synthetic small-molecule chemistry.</p><p>So
  I''m going to talk about nature''s ligands today at sort of two levels of size.
  Going to talk first about the porphyrin ligand. And then next, we''re going to talk
  about proteins.</p><p>OK. First this porphyrin ligand is one of the most pervasive
  small-molecule hosts for metal ions within biomolecules. And it''s really quite
  a remarkable ligand type. And I''d like you to appreciate it at the level of being
  able to draw the structure of it and to see just how this system can provide a host
  for metal atoms.</p><p>It is a system that has four nitrogens that it can donate
  to a metal center. So it''s a tetradentate ligand. We talked about bidentate ligands
  earlier in the semester. This is a tetradentate ligand that nature uses to hold
  metal ions very tightly where it wants them in a biomolecule.</p><p>And first you
  draw the four nitrogens at the corners of a square. And now we''re drawing in the
  carbon parts of the molecule. You can draw four vertical lines like that and four
  horizontal lines like this. And then what we''re going to do is, at each of the
  four corners of this square-shaped molecule, we''re going to go ahead and complete
  five-membered rings. Like that.</p><p>And then finally, we have four more carbons
  to add to this structure. And those four carbons are at the top, bottom, and left,
  and right, like this. OK.</p><p>So you can see that the porphyrin ligand is a large
  ring-shaped macrocycle. And where can a metal ion go? It can go right in the center.</p><p>If
  you are going to figure oxidation states that involved a porphyrin molecule, then
  you would need to know the charge like we know that one chloride is in a metal complex.
  It carries a one minus charge. And you need to know how many charges the porphyrin
  ligand would carry if a metal were in there, so that you could figure the oxidation
  state and the d-electron count for the metal that would be inserted into the porphyrin.</p><p>And
  to do that, we first recognize that if you isolate a porphyrin without the metal
  in there, then two of the four nitrogens have hydrogens attached. OK? And also this
  is a system-- you''ll remember the graphite structure that we looked at last time
  that had all these p orbitals perpendicular to the plane of the molecule. And similarly,
  this is an unsaturated molecule that has lots of pi bonding going on with the carbon
  p orbitals that are perpendicular to the plane of the blackboard here. So it can
  be a quite flat molecule, but as you''ll see it can ruffle as well when it carries
  out its biological function.</p><p>And so we''ll go ahead and draw in the double
  bonds. And this is the part where people sometimes get messed up because one thing
  we don''t want to do is draw five bonds to a carbon atom. OK? And if you see a structure
  that''s drawn that way, you know that someone has made a mistake because they violated
  the octet rule. All right?</p><p>So let''s draw in, starting here at the upper left,
  we''ll put in a double bond here, and here, and here. And then this is where it
  gets interesting because I could put a double bond here now or here because this
  carbon''s going to need a double bond. I''m going to choose to put it here, and
  then I''m going to use symmetry across the way to put one there.</p><p>And having
  done that, I can now go here, here, here, here, and-- so I didn''t use symmetry
  across the way. This is much easier to do and it''s small. OK.</p><p>So now you
  can see that these two nitrogens here would have lone pairs that point into the
  center of the porphyrin ligand and same across the way by symmetry there. And then
  when we remove these two hydrogens as H plus and insert a metal ion, you''ll see
  that the porphyrin is actually behaving as a dianion. When a metal ion and-- most
  typically, we''re going to see that the metal that appears at the center of a porphyrin
  ligand is an iron center.</p><p>And so the nitrogens provide four lone pairs in
  the plane of the molecule that are directed at the metal center. So that''s four
  out of the six positions for an octahedron all in the same plane. And because of
  the rigid delocalized structure of the porphyrin ligand, what we find is that a
  metal center that is in a porphyrin is very difficult to take out of the porphyrin
  because these lone pairs are forced at all times to converge on the same point which
  is the metal ion.</p><p>OK. You know that I enjoy giving lectures using shock. But
  sometimes when the molecules are really big, it''s easier to use a computer to portray
  them quickly. And that''s what we''re going to do today, assuming my computer participates
  and cooperates with me today.</p><p>And what I really want to do is to go through
  and show you a tour of six really important metalloproteins. And all of these, you''ll
  see the heme plays an important role, the iron in a porphyrin. This unit together,
  when an iron is located inside of a porphyrin, that is referred to as heme. So you
  can have other metals in there, and then it''s not called a heme anymore, but that
  would be very similar structurally.</p><p>OK. So I''m going to take advantage of
  some really nice summaries that have been put together on the protein data bank
  website. So since the first crystal structure was determined for a protein, the
  structural information has been collected and put together in one site so that researchers
  all over the world can take advantage of this information to advance science and
  medicine. And since 2000, the protein data bank has been collecting information
  about certain biomolecules and putting out one new little short story or vignette
  each month. So you have their Molecule of the Month portion of the protein data
  bank website. So each month, you can go and look and see which new molecule has
  been added and learn more about it.</p><p>And I want to start out here with myoglobin.
  OK. And I''m going to give you this website information online associated with today''s
  notes. So you''re going to see that myoglobin was the first protein structure.</p><p>There
  are many different ways to represent protein molecules. Protein chains are composed
  of linear polimers of the naturally occurring amino acids with polypeptide chains.
  And they can sometimes be very, very large in length, and then they fold up and
  make three-dimensional shapes. And these are big ligands for metal ions. And what
  you''re going to see is that in many of the most important proteins, the place where
  the reaction''s actually happened is that the metal center that''s embedded somewhere
  in the protein chain as it has folded up around it to modulate function.</p><p>OK.
  So this is a pretty small protein-- myoglobin is. It''s an oxygen storage protein.
  So as you know, diatomic molecules like dioxygen are important for life on Earth,
  and O2 in particular-- not only do we have hemoglobin that we''ll talk about next,
  but there are also oxygen storage molecules like this myoglobin molecule. And this
  was the first structure that was determined, and that happened in 1960.</p><p>So
  we''ve been getting information at the molecular level about how biomolecules work
  really since 1960 using x-ray diffraction techniques. So first, a worker in this
  area has to obtain a large quantity of the protein in question, get it purified,
  and then find conditions under which it will grow single crystals so that you can
  impinge upon it with an x-ray beam and then looked at the intensities of the diffracted
  x-ray beams and then work back to solve the crystal structure and find out what
  arrangement of atoms in 3D space could have given rise to that pattern of diffraction
  intensities. And that was first done for a protein in 1960.</p><p>You might find
  it interesting that this protein was isolated from sperm whales. OK? Sperm whale
  muscles. And in order to get a protein pure, as I said, you need a lot of it. And
  the muscles of sperm whales are very large indeed, and so these workers were able
  to collect large amounts of this red protein myoglobin from sperm whale muscles
  in order to finally get enough in pure form to crystallize and solve the crystal
  structure.</p><p>And since this is involved in oxygen storage and you know that
  whales and other marine mammals can dive below the ocean surface for long periods
  of time, they need to store lots of oxygen so that they can use their muscles during
  the length of a long dive. And this protein is very abundant in muscles. It gives
  meat its red color and so forth. The color is coming from the heme units that are
  embedded in the myoglobin. And I''ll step through a little bit of this, but I would
  like to actually show you some more of the features of this.</p><p>You''ll see as
  you go through this website, there are both static and interactive ways to look
  at metalloproteins. And sometimes the polypeptide chain composed of the amino acids
  is just represented here as sort of a tube, so you can see the polymer has wrapped
  up. And here they''ve chosen to represent the heme unit as spheres to highlight
  it and accentuate it.</p><p>And in this first structure of myoglobin, there was
  not an O2 molecule bound to the heme unit. Instead, there was a water molecule bound
  to the iron center of the heme unit. And then down here, they rotate the structure
  perpendicular so you can appreciate the square nature of the heme unit together
  with when viewed edge on, its flat nature. So that''s the structure here. And you
  can see how this myoglobin molecule, this big protein polymer, has folded up like
  a clam to grasp this heme unit that inside of it will bind O2 and store it when
  oxygen comes to it.</p><p>And so later structures of myoglobin have been carried
  out that do show you the position of the O2 molecule. And in these two representations
  that we are given here, we first see, once again, the heme unit as spheres and the
  protein side chain just as tubes. And then this representation in the bottom, all
  the protein side chain atoms themselves are represented as spheres. And you can
  see that when that''s the case, you can tell that the atoms of the protein ligand
  really fill up space very well. And what they''re showing you with this circle here
  is the location beneath that side chain of the dioxygen molecule inside there.</p><p>And
  so they''re trying to give you the idea that when O2 is on the heme unit inside
  the myoglobin, the myoglobin protein matrix actually covers that up. And so in order
  for the O2 molecule to get into the heme and also to exit from the metalloprotein,
  this thing has to be flexible enough to open up and allow the dioxygen molecule
  to exit from its chamber inside the metalloprotein.</p><p>OK. Now next, I would
  like to go over to the site where we''re actually able to look at things interactively.
  And you need to be able to enter the code for the protein in question.</p><p>And
  so this is a structure of the myoglobin molecule that was obtained at molecular
  or atomic resolution. So one of the parameters that you''ll see when people discuss
  protein structures is the resolution to which the data have permitted the structure
  to be determined. And the best quality structures, the ones where we know most precisely
  where the atoms are, are the ones that have very low resolution numbers. In other
  words, this one, if you looked at the introductory web page there so that the resolution
  was at the level of one angstrom.</p><p>OK? So we''re talking on the order of carbon-carbon
  bond distances so that you can actually see all the individual atoms quite clearly.
  And this has to do with the quality of the crystal and how large was the angle at
  which you could collect spots from the diffracted x-rays as they came off of the
  crystal.</p><p>And so this is nice. This one requires Java. That''s why I was not
  able to use Athena today. But actually, I come in here, and I see that the Athena
  terminal is down anyway, so this was a good choice as it turns out. And this is
  nice because you can actually zoom on these structures.</p><p>One thing you notice
  is that this was crystallized with sulfate present in the medium, so there was some
  buffer present. And it had sulfate, and the sulfates crystallized together with
  the protein chains. And as we look at representations of proteins, you''ll appreciate
  that this representation shows the polypeptide backbone as these ribbons. And that
  emphasizes that when proteins fold and go into a three-dimensional shape that they
  need for their function, they can usually do so either by making coils like this
  that look like springs-- these are called helices. Or they can just form random
  coils as you can see out here.</p><p>And oftentimes you have these helices that
  are like long springy tubes that are then connected at their termini by coils of
  the protein side chain. And then, of course, here embedded within this structure
  is our heme unit. And one thing if you look at this one pretty closely, you''ll
  be able to see that this heme is a little more interesting, a little more complicated,
  than the one that I drew on the board here because the one I drew on the board was
  assumed to have just hydrogens at each of these carbon positions at the periphery
  of the molecule.</p><p>But when we look at the one that''s actually been found in
  myoglobin at atomic resolutions-- so you see this one is actually with the O2 molecule
  present. For some reason, no bond drawn there, but that''s OK. We''ve talked a lot
  about molecular representations, and we understand what''s going on here. It''s
  that people have decided that we would represent the atoms over here without any
  detail at all just showing them as ribbons.</p><p>And then here we''re going to
  look at the atom positions. And you can see the five-membered ring here, here, here,
  and here at each nitrogen corner of the heme unit. There''s the iron center.</p><p>And
  then, notice that the substituents, instead of being just hydrogen or organic, there''s
  a carbon here. This looks like a methyl group perhaps. And this one looks like an
  ethyl group or maybe a vinyl group here if this is one of the etioporphyrin ligand
  types.</p><p>And then coming out here, we have CH2, CH2. We have something that
  looks like it might be a ketone residue since we have a single oxygen on a carbon
  there, or alternatively, it could be some other oxygenated residue. But over here,
  you see that there''s a link to this five-membered ring of the heme. You have a
  couple of methylenes and then a carboxylate residue.</p><p>And so when you investigate
  these structures, these metal complexes that are embedded in the protein usually
  are held in position, both by the way the protein folds up around the heme unit,
  but also because, in this case, the heme unit is connected to residues like this
  carboxylic acid residue that can hydrogen bond to specific residues on the protein
  side chain. So if we were to look at the atoms involved in the side chain right
  over here that''s next to this carboxylate residue, this probably has some hydrogen
  bond acceptor that is interacting with this carboxylate residue-- and same over
  here-- to hold this heme unit in its desired position or in its required position
  for function. So please keep that in mind as you go ahead and look at different
  structures in the database.</p><p>And now let''s go to the 2003 description of hemoglobin.
  This is rather nice. OK.</p><p>So myoglobin was a pretty small protein having a
  single heme unit. And hemoglobin is much more interesting. It actually has four
  big pieces that come together, and it has four heme units in it.</p><p>OK. And as
  you go through this website, you''ll find out about why blood is red versus blue.
  It''s actually red in the oxygenated form and blue in the deoxygenated form. And
  there are many crystal structures of different hemoglobins and mutants of hemoglobins
  in the database and both with and without the oxygen molecules.</p><p>There''s a
  discussion here of artificial blood. And as you''ll see what the premise is here
  for the design of artificial blood motivated by a desire to have blood available
  for transfusions and so forth that would not be contaminated, would be just synthetic--
  would be just pure hemoglobin in fact. But what happens is that without the protective
  casing of the red blood cell, the four parts of the hemoglobin molecule fall apart
  from each other, and they don''t do their function properly anymore.</p><p>And we''ll
  learn more about the function in a moment. So people have actually made mutants
  wherein you make covalent bonds between the four different subunits of the hemoglobin
  so that they can''t fall apart. And that''s one of the approaches to the synthesis
  of artificial blood.</p><p>And then this is a rather nice little video which comes
  from two different hemoglobin crystal structures. If you imagine different crystal
  structures, like crystal structure of hemoglobin when it has O2 bound verses when
  it does not have O2 bound, as snapshots of the molecule in motion, then you can
  generate a little movie like this one shown here from the experimental data on the
  crystal structure. And so the really remarkable thing here is that-- see when the
  O2 molecule is bound to the heme, then the protein itself, this large unit that
  is comprised of four parts, as one confirmation, one structure, and it changes quite
  a bit. You see how much it is moving around when the dioxygen molecule comes off
  of the heme. And that is related very much to its function.</p><p>In fact, it turns
  out that, in hemoglobin, when the deoxyhemoglobin arrives at a place in your body
  where there is a lot of oxygen, the first O2 molecule binding event is very difficult
  to achieve because it involves a big change in the structure of the entire protein
  molecule. But once that first O2 molecule has bound, the whole protein molecule
  has changed its structure such that the next three hemes take up their O2 molecules
  rapidly in rapid fire succession after the first one has done it.</p><p>So the heme
  goes into a place of high-oxygen concentration and it rapidly loads up with four
  O2''s. And then it moves on. And then when it gets to a location that needs oxygen,
  the first one comes off, and then the next three fire off in rapid succession because
  of the change in the protein shape. And so hemoglobin is, therefore, very good at
  collecting four O2 molecules at a time and then delivering them to where they''re
  needed.</p><p>And this is just a beautiful example of the way protein crystallography
  can teach us about mechanisms of biomolecules. OK. And then also you can learn about
  disease mechanisms in the data bank here because this is from a different crystal
  structure where a mutation has occurred in the protein side chain of the hemoglobin.</p><p>So
  this is one hemoglobin molecule. And it turns out that when that mutation is present,
  unfortunately, it causes hemoglobin molecules to aggregate and stick together and
  clump together like that. And that phenomenon is associated with diseases like sickle
  cell anemia. OK?</p><p>So the structure of the protein is changed when one residue
  of the amino acid side chain is changed to a different amino acid. And then these
  things all aggregate together instead of forming nice disks as normal red blood
  cells do. They stick together and form these different shapes that are not good
  at functioning the way they''re supposed to.</p><p>And then here is a close-up on
  where the action takes place in the hemoglobin molecule. And this, again, is from
  the two crystal structures that we were talking about-- the one that has the oxygen
  molecule bound to the iron of the hemoglobin and the one that does not. And one
  thing that you''ll notice is that when the O2 molecules is absent, this iron in
  yellow represented here as this yellow sphere is sort of dipping down on one side
  of the hemoglobin-- the heme plane toward the nitrogen of this histidine. So this
  is part of the amino acid protein side chain. And a histidine residue has this Lewis
  space group here that is coordinated to the iron.</p><p>And so in that form, this
  is a five-coordinate metal center with five ligands of square pyramidal geometry.
  It''s an octahedron missing one of those six ligands. And then when the O2 ligand
  comes in and binds and becomes the sixth ligand in the coordination sphere of the
  iron, the iron responds to that by moving up to the other side of the porphyrin
  plane, and it draws with it the histidine ligand.</p><p>So the iron moving up pulls
  the histidine up. And that structural change at the iron center is then propagated
  throughout the amino acid side chain to which this is connected and then, ultimately,
  throughout the entire protein to give you that huge conformational change that we
  saw when O2 is bound versus when it''s not. So the coordination chemistry of the
  iron here is really controlling the peptide confirmation and the function of this
  protein.</p><p>OK. So these bio molecules actually accumulate quite a lot of the
  concepts that we''ve talked about all semester. And that phenomenon is going to
  continue as I take you through a tour of a couple more of these important molecules.</p><p>Now
  the first one was a small one that we looked at, myoglobin, and I''m going to continue
  that sort of progression by now talking about another small protein. This one may
  even remind you in its appearance of myoglobin. But instead of serving to store
  O2, like myoglobin, cytochrome c, which has a single heme unit in it here, serves
  as a shuttle for electrons. OK?</p><p>The pathway that electrons take in the whole
  process of respiration in organisms is really quite fascinating. We require reducing
  agents. We eat food. Those are reducing agents. And then we go ahead and burn that
  fuel using the oxygen molecule.</p><p>And that''s where we get our energy. We don''t
  burn the fuel literally by having us burst into flames, but we do it in a controlled
  manner, in a stepwise manner. And we take advantage of those steps to carry out
  the processes of life.</p><p>And so there are proteins like these small cytochrome
  c proteins that are developed expressly for the purpose of moving electrons from
  one place in the body to another. So it''s like a little electron shuttle protein.
  And these are just two views of it.</p><p>And if you read this part of the website,
  you''ll also learn that this one has remained virtually unchanged over eons. It''s
  a very ancient protein. OK.</p><p>And this picture shows you cytochrome c protein
  and where it picks up its electrons and then where it goes to drop them off. So
  here''s an enormous protein relative to cytochrome-- cytochrome c is the little
  teeny one here with just a single heme unit. OK?</p><p>Look at this thing. This
  thing is-- I don''t know what this looks like, but it looks like a bizarre-shaped
  entity, shall we say. And this yellow stripe that goes across the page here is actually
  a membrane. So proteins are often classified as to whether they are membrane proteins--
  meaning that they don''t move around. But they''re embedded or fixed in a membrane
  with one part of their molecule on one side of the membrane and the other part of
  the molecule up on the other side of the membrane and then the part of the molecule
  that''s actually located within the membrane, and then non-membrane proteins like
  cytochrome c that are actually mobile and can move about within a cell. And cytochrome
  c goes over to this large protein which has lots of different cofactors shown here
  in red which incidentally are hemes themselves.</p><p>And so this molecule-- the
  big protein here embedded in the matrix is called cytochrome bc1. And it is a protein
  involved in this process of respiration. It is producing electrons. And cytochrome
  c comes over here, and as you can see, it fits really well into this spot right
  here on cytochrome bc1.</p><p>And that''s a feature of protein-protein interactions
  that they often contain residues on their surface that are hydrogen bonding residues,
  for example, that make their surfaces complimentary. And it means that when cytochrome
  c comes in into cytochrome bc1, it actually locks in in one particular way. And
  this complex between cytochrome c and cytochrome bc1 has been crystallized. And
  the crystal structure of it''s in the database, and that''s where we get this representation
  shown here. So you can actually go through-- if you''re a crystallographer-- and
  look at all the different complimentary interactions that holds cytochrome c into
  place on cytochrome bc1 when it docks there.</p><p>And what I''m going to tell you
  in a moment is that as electrons are originating within cytochrome bc1 here, they
  have a pathway through which they move and they get up into this heme unit. And
  then when cytochrome c docs, the heme unit within it can get close enough to this
  heme unit that their wave functions can overlap, and the electron can tunnel right
  across into the cytochrome c which accepts it. And then it goes off, and it''s looking
  for this protein next which has also been crystallized and characterized in the
  database. And we''ll talk about it next. This is called cytochrome C oxidase because
  it oxidizes the reduced form of cytochrome c.</p><p>And let me just say that probably
  one thing we won''t have on the final will be working the molecular orbital diagram
  of this molecule. As you can see, we''ve got up to a position here where the number
  of orbitals is kind of prohibitive to do that sort of thing. Now you can understand
  why the computational chemists have quite a big challenge as they try to understand
  at a molecular level all of the processes of life and, really, a long way from doing
  that actually.</p><p>When people attempt to do that these days, they''re usually
  making some big approximations with the protein side chain because that''s the electronically
  uninteresting part of the system. Where the metals are is where everything is happening.
  Let''s go down here and see what else we can learn.</p><p>OK. Here''s a close-up
  with a different representation of when cytochrome c molecule docs on to the cytochrome
  bc1 complex. And here we''ve got the polypeptide chain of cytochrome c represented
  as these pink tubes, and then as these yellow tubes down here, this little part
  of the cytochrome bc1 molecule that is reaching up to interact with cytochrome c.
  And here''s the heme unit of cytochrome bc1 that is right at the surface of that
  protein so that it can overlap it''s wave function with the heme unit of cytochrome
  c to permit an electron to transfer and reduce cytochrome c.</p><p>OK. And so now
  we''ll go on to cytochrome c oxidase. OK.</p><p>So in reading about cytochrome c
  oxidase, you''re going to learn more about oxygen. And this piece of today''s story,
  together with the one that I''m going to go to next, which will be with respect
  to photosystem one and photosystem two, are sort of the two ends of the chain of
  respiration. Because with photosynthesis, that we''ll talk about next, we use light
  energy to create O2. And most organisms on the planet have these cytochrome so that
  they can use O2. In fact, what they do is they take electrons that are derived from
  food and get energy by combining them with oxygen, reducing them-- reducing the
  oxygen molecule-- to water, and at the same time pushing protons across a membrane
  which stores energy that can be used for other processes like building up ATP.</p><p>And
  so, the cytochrome c oxidase is getting its electrons from cytochrome c and it''s
  using those electrons to reduce the O2 molecule. And for that reason, cytochrome
  c oxidase should have somewhere in it a place for the dioxygen molecule to bind.
  Just like is the case for hemoglobin. We found out where oxygen binds in hemoglobin.
  And notice that as you go through they''ll make an analogy here to charging of a
  battery or actually of a capacitor as you''re reducing the O2 molecules with these
  electrons coming in from the cytochrome c shuttle, and making water and pushing
  protons across a membrane that is being likened to charging a battery or charging
  a capacitor.</p><p>And this is a membrane-bound protein just like the bc1 complexes.
  And here''s a nice graphic that shows a number of the important cofactors. A cofactor
  is just something that''s an integral part of a protein but isn''t part of the amino
  acid backbone of the polypeptide chain. And usually it''s something like a heme,
  but here not only do we have hemes, but here''s a heme that is thought to be involved
  in the electron transport chain within the cytochrome C oxidase molecule. And over
  here is a heme that is thought to be important in binding the O2 molecule. And also
  this one has copper.</p><p>So these 3D transition elements are turning out to be
  pretty important in biology. The iron is frequently the site for the binding of
  a diatomic molecule, but look at this. This structure that''s in the Cambridge database
  here, its PDB entry 1OCO, actually has a carbon monoxide ligand bonded to the iron
  in the active site where O2 is supposed to bind when this enzyme functions. So this
  is a poisoned form of the enzyme. CO comes in and binds and shuts this enzyme down,
  and that''s what they were able to purify and crystallize and get the 3D structure
  of.</p><p>And then here you have a heme iron which interacts with the carbon end
  of carbon monoxide. And then positioned over here, and also ligated by residues
  from the protein side chain, is copper, referred to as copper b. So this is a copper
  ion. And that means that when a diatomic molecule goes in and binds to the heme
  at one end, it''s other end is binding simultaneously to copper b. And that diatomic
  molecule is serving as a bridge between two metal ions-- the iron heme and the copper
  b up here.</p><p>And then also up here is a copper a site, where you have a pair
  of coppers. And this site is referred to and thought to be the port of entry of
  the O2 molecule into cytochrome c oxidase. That O2 somehow comes in and may bind
  here first, but then go over here and electrons are coming in through this other
  heme unit into the one that binds the O2 and ends up reducing it to generate water.
  So that is a really pretty picture of how complicated the machinery of an enzyme
  can be to take advantage of all this lovely transition element chemistry and redox
  chemistry to carry out life function.</p><p>And also-- let''s see-- OK, so what
  they''re doing here is analyzing this. This cytochrome c oxidase is composed of
  a number of different protein chains that are all packed together into the overall
  complex, and they''re coloring each of these residues a little differently here.
  And then when you get down to the bottom of this page, they''re comparing a subunit
  of cytochrome c oxidase to an actual cytochrome c oxidase that is found in bacteria
  molecules. Bacteria organisms. So bacteria have their own cytochrome c oxidases
  and this looks like the central core of our cytochrome c oxidases.</p><p>And the
  idea is that mitochondria in cells may, at some point way, way back in evolution,
  have been the result of bacterial invaders into cells. And then these bacteria became
  integral parts of ourselves, and we added new proteins on and elaborated to change
  the function as we deemed necessary through evolution. But these ways of comparing
  structures of simpler or ancient proteins to ones that are more modern is an interesting
  way to learn about how life has evolved on this planet.</p><p>OK, and-- so cytochrome
  c oxidase is a pretty remarkable protein. And I want to see if I can get this thing
  up interactively because the structure is quite impressive. And I''m using this
  carbon monoxide substituted variant. When these structures have many thousands of
  atoms, sometimes these things take a little while to load. So hopefully this won''t
  take too long.</p><p>The viewer that-- there are actually several viewers that you
  can choose from on this protein data bank web page, and they possess features that
  allow you to turn on or turn off different parts of the molecule. You can subtract
  away the cofactors, or subtract away some of the loops or the helices, and that
  allows you to explore the structure in quite some detail. Now, each of these are
  different protein residues, different residues of this enormous protein. So let''s
  come down here, zoom in a little bit. OK, if I notice that-- there I can subtract
  away the coils that hold together the various loops, OK? Or you can take away, in
  fact, all of those things.</p><p>And notice the cofactors themselves are these heme
  units and they have some interesting and complicated side chains. But they''re the
  same sort of heme units in general that we see in myoglobin or in cytochrome c.
  OK? Let''s see . Actually-- there are orientations of this molecule that you can
  access that shows you how this thing fits nicely into a membrane. This is one of
  them. You see all these different helices that are aligned with one another.</p><p>This
  is the part of the cytochrome c oxidase that resides in the membrane. And all of
  these residues are lipophilic, so they interact with the non-polar interior of the
  membrane very well allowing this protein to just stay in as part of the membrane.
  And then these loops that extend on one side of the membrane or onto the other contain
  polar residues that would prefer to interact with water or with charged things in
  solution. And so that''s an important aspect of the way a membrane protein like
  this is the structured. If I could just take away the loops here, the coil, that
  becomes quite clear.</p><p>So now, with the few minutes we have left, I''m going
  to go ahead and talk about photosystem I and II at the other end of the chain of
  respiration where organisms are actually-- you know, they realize that eventually
  you would run out of all the food on the planet and everything would die. And so
  they learned to absorb light and to use light energy to carry out processes of life
  instead of just using the energy that you get when you add electrons to water, to
  two oxygen and make water. And so let''s go look first at photosystem I.</p><p>Photosystems
  I and II are different parts of the photosynthetic pathway and of the electron utilization
  chain, electron transfer chain. And what you see is that photosystem I and II are
  both going to be membrane proteins. These, incidentally, are usually much harder
  to obtain in large quantities and much harder to crystallize, and their structures
  much harder to characterize. So there are a lot of important membrane proteins whose
  structures are not very well known yet. In photosystem I it''s a trimer. There''s
  one piece right here, it''s shaped like a big disk. Here''s another piece and here''s
  another piece. And then if you flip it by 90 degrees to look at it edge-on you can
  see here it is sitting in the membrane.</p><p>And these have colorful cofactors.
  These now have porphyrins, many porphyrins, that contain instead of iron, contain
  magnesium. And these are the chlorophylls that are present in here as light gathering
  entities. And then-- OK. OK, with this picture I can tell you quite a bit about
  photosystem I. Here, you can see, is the part of the molecule that is embedded in
  the membrane. And what we have is the ability to extract electrons from a molecule
  that is hard to oxidize.</p><p>OK, so in photosystem II you''ll see that the source
  of electrons is water. And so photosystem II is capable of pulling electrons out
  of water, generating dioxygen, the most important photosynthetic reaction on our
  planet. In the case of photosystem I, the source of the electrons, instead of being
  water, is a little protein called plastocyanin. So plastocyanin comes up here and
  it is oxidized. And it''s a little bit like the LED we talked about last time, because
  what happens is there''s a pair of heme units here, they''re not hemes of chlorophylls,
  the special pair, that when light energy is absorbed and excitation occurs, it sends
  an electron to high energy.</p><p>And normally, after such an absorption of light
  occurs, that electron would just drop back down and the molecule would go back to
  its ground state. But what this molecule has been built to do is to take that high
  energy electron and descend it through a series of redox active molecules. And it
  actually goes all the way through here, and these are little iron four s four cubes.
  So these are beautiful little inorganic molecules, Fe4 S4 cubes. The electron pathway
  goes through these redox active units and jumps from one of these Fe S4 cubes to
  another, and finally jumps out to a protein that docks up here to accept it, which
  is a ferredoxin protein.</p><p>So after light comes in and you get a high energy
  electron, you get a hole, the hole is filled through oxidation in the case of photosystem
  I and plastocyanin in the case of photosystem II of water, and so the chain goes.
  Now, having run out of time here, I will talk about photosystem II at the beginning
  of next hour.</p>'
type: course
uid: e23e92da4e7decf55b9a4e6c8cba8d01

---
None